frontend/src/components/DeleteTripButton.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+
+export default function DeleteTripButton({ tripId, userId, onTripDeleted }){
+    const [error, setError] = useState(null);
+    const [success, setSuccess] = useState(false);
+
+    const handleDelete = async () => {
+        // Pop-up window confirms if user wants to delete trips
+        const confrimDelete = window.confirm("Are you sure you want to delete this trip?")
+        if(!confrimDelete) {
+            return;
+        } 
+
+        try {
+            // Makes a DELETE request
+            const response = await fetch(`http://127.0.0.1:5000/delete_trip`, {
```

Comment:
Do we want to parameterize the beginning of this URL so it can easily be accessed and/or changed in an .env file rather than hard coded here? That way, when we host this we won't have to change any code.